src/dev/riscv/plic.cc

```diff
@@ -415,7 +416,7 @@ Plic::propagateOutput()
     }
 
     // Add new output to outputQueue
-    Tick next_update = curTick() + cyclesToTicks(Cycles(3));
+    Tick next_update = curTick() + cyclesToTicks(Cycles(outputLatency));
```

Comment:
Could this break things if `outputLatency` is 0? If so, we should either have a fatal in the constructor.

I'd also suggest the following to make sure that you're getting a clock edge. The current code could return a tick that is not on a clock edge if curTick() is not on a clock edge.

```suggestion
    Tick next_update = clockEdge(Cycles(outputLatency));
```

Reply:
> Could this break things if `outputLatency` is 0? If so, we should either have a fatal in the constructor.

No, we can schedule the intterupt output greater eqaul than current tick
https://github.com/gem5/gem5/blob/f93c153f2c2d7d5b3266054fdf421749e9589783/src/sim/eventq.hh#L757-L760

Reply:
Sure, it makes sense that it's OK to do on the event queue. I was more worried about the PLIC logic of having two interrupts in the same tick. This could cause problems in other logic. Or, it could be no problem at all. I'll resolve this issue.